openshift-knative-operator/cmd/operator/kodata/knative-serving/0.19.0/5-serving-domainmapping.yaml

```diff
@@ -236,6 +243,13 @@ spec:
               weight: 100
       serviceAccountName: controller
       containers:
+        - name: kube-rbac-proxy
+          image: quay.io/brancz/kube-rbac-proxy:v0.7.0
```

Comment:
Is this image overridable via the CSV? We have to replace this for productization with a productized image.

Reply:
@markusthoemmes afaik this is the productized version of rbac-proxy, at least this what I was suggested to use? @s-urbaniak  can you verify this?

Reply:
I will make it configurable.

Reply:
I will make it configurable sure.